src/Covid19.Web/Services/APIService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Covid19Api.Models;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Covid19Api.Services
+{
+    public class APIService : IAPIService
```

Comment:
I would consider changing the name of this class to something more relevant like "CovidService" instead of APIService